content/articles/userstory-techstory.md

```diff
@@ -0,0 +1,169 @@
+
+In the context of our work at the CTL, these user stories inform us developers
+what the software should do to satisfy the learning objectives, and how it can
+yield the specified outcomes. We gather these user stories to define the sets
+of functionality of the learning tools we’re building.
+
+These user stories alone are not enough for us to begin implementing the
+functionalities. As simple as they may sound, complexity can happen from the
+point of view of the system, or technology, and it can be hidden without
+further examination. Sometimes this system’s perspective introduces a chain of
+complications that can change user experience, and perhaps affect the learning
+objectives outcomes.
+
+This is why I write technical stories.
+
+## What are technical stories?
+
+There are
+[many](https://rgalen.com/agile-training-news/2013/11/10/technical-user-stories-what-when-and-how),
+[many](https://mentormate.com/blog/when-user-stories-get-technical/)
+definitions and
+[guidance](https://medium.com/tribalscale/writing-technical-user-stories-434bf96f1dd5)
+on writing technical stories, and even
+[disagreements](https://www.extremeuncertainty.com/why-technical-user-stories-are-bad/)
+on their usefulness. A technical story follows along a user story, coupled with
+[acceptance criteria](https://www.freecodecamp.org/news/the-acceptance-criteria-for-writing-acceptance-criteria-6eae9d497814/),
+from the point of view of the system that will be developed or used to
+implement the functionality.
+
+Technical stories are not in lieu of user stories. In fact, they add value to
+the specification initially defined by user stories. They also clarify backend
+conditions for acceptance criteria. It’s noted that technical stories are not
+instructions on how developers should implement the functionalities. They’re
+narratives on what happens next in the system, and guides the technical
+specifications.
+
+As a front-end developer, I find them extremely valuable in designing and
+refining the software’s user experience, communicating with the project’s
+developers and stakeholders, drafting out specifications, and sketching out the
+wireframes.
+
+## Why write technical stories?
+
+User stories are invaluable to define user expectations of a desired
+functionality in simple and direct manner, but they may not make apparent the
+details and conditions that may come up on the backend. Writing out these
+technical narratives defines the scope and helps us estimate our efforts and
+prioritize the functionalities.
+
+Technical stories are also a form of conversation among us developers and
+designers, and the project’s stakeholders. These stories are not what and how
+we developers want the software to do, but they are an examination of options
+and possibilities to meet the acceptance criteria, and satisfy the learning
+goals and outcomes. We may not have thought of these potentials in detail as we
```

Comment:
Sentence beginning "These stories are not..." suggest breaking it into two sentences.